<launch>
    <node pkg="tams_tactile_sensor_array" type="tactile_sensor_array.py" name="TactileSensor" output="screen">
        <rosparam command="load" file="$(find tams_tactile_sensor_array)/config/params.yaml"/>
    </node>
</launch>
